FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -18,7 +18,14 @@ import Foundation
 @_implementationOnly import FirebaseCoreExtension
 @_implementationOnly import FirebaseInstallations
 @_implementationOnly import GoogleDataTransport
-@_implementationOnly import Promises
+
```

Comment:
Fix  /Users/runner/work/firebase-ios-sdk/firebase-ios-sdk/FirebaseSessions/Sources/FirebaseSessions.swift:21:22: warning: using '@_implementationOnly' without enabling library evolution for 'FirebaseSessions' may lead to instability during execution